app/src/interfaces/input-rich-text-html/input-rich-text-html.vue

```diff
@@ -246,7 +246,6 @@ const props = withDefaults(
 		],
 		font: 'sans-serif',
 		customFormats: () => [],
-		disabled: true,
```

Comment:
Unrelated to this issue, but I don't think we should explicitly set the `disabled` value in the interface.